project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -39,20 +46,22 @@
 
     /** In this radius around "LOCATION" places will be searched. */
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
     /** The entry point for a Google GEO API request. */
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    /** A mapping between cuisines and text search words. */
+    private static Map<String, String[]> cuisineToSearchWords = getCuisinesMap();
```

Comment:
why do we keep this outside if it's only used in one place?

Reply:
I wanted to read the json and create the map once and reuse it for all queries (instead of calling the getCuisinesMap() on every query). Do you think it is better to do this differently? 
Anyway I changed it to have constant name.

Reply:
Yeah, if you'll use it multiple times that'll make sense.